committeeoversightapp/templates/partials/rating_table.html

```diff
@@ -1,37 +1,45 @@
 <div class="table-responsive">
-    <table class="table table-borderless table-striped text-center">
-        <thead>
-            <tr>
-                <td></td>
-                <td colspan="2" class="font-weight-bold">
-                    Committee Performance
-                    <hr class="mb-0" />
-                </td>
-                <td colspan="3" class="font-weight-bold">
-                    Number of Hearings
-                    <hr class="mb-0" />
-                </td>
-            </tr>
-            <tr>
-                <td></td>
-                <td>CHP Score</td>
-                <td>Grade</td>
-                <td>Policy/Legislative</td>
-                <td>Investigative/Oversight</td>
-                <td>Total Hearings</td>
-            </tr>
-        </thead>
-        <tbody>
-            {% for rating in page.committee.committeerating_set.all %}
-            <tr>
-                <td>Congress {{ rating.congress }}</td>
-                <td class="{{ rating.css_class }} font-weight-bold">{{ rating.rating }}</td>
-                <td></td>
-                <td></td>
-                <td></td>
-                <td></td>
-            </tr>
-            {% endfor %}
-        </tbody>
-    </table>
+  <table class="table table-borderless table-striped text-center table-sm">
+      <thead>
+          <tr>
+              <td></td>
+              <td colspan="3" class="font-weight-bold">
+                  Number of Hearings
+                  <hr class="mb-0" />
+              </td>
+              <td colspan="2" class="font-weight-bold">
+                  Committee Hearing Performance
+                  <hr class="mb-0" />
+              </td>
+          </tr>
+          <tr>
+              <td></td>
+              <td>Investigative/Oversight</td>
+              <td>Policy/Legislative</td>
+              <td>Total Hearings</td>
+              <td>Score</td>
+              <td>Grade</td>
+          </tr>
+      </thead>
+      <tbody>
+          {% for rating in page.committee.committeerating_set.all %}
+          <tr>
+              <td>Congress {{ rating.congress }}</td>
+              <td>TK</td>
```

Comment:
Is there a way to vertically align these cells to the middle? They're pulled to the top at the moment.

<img width="1009" alt="Screen Shot 2019-10-28 at 3 40 26 PM" src="https://user-images.githubusercontent.com/12176173/67716158-48e83200-f999-11e9-8827-9df802a16c09.png">

Reply:
good eye! yes